frontend/src/main.ts

```diff
@@ -8,6 +10,8 @@ import Lara from "@/presets/lara";
 import router from "@/router";
 
 const app = createApp(App);
+app.use(ConfirmationService);
+app.use(ToastService);
```

Comment:
Do we have some configuration options? Can we make the Cancel button visually different? I.e. making it grey and to the left side?

Reply:
I changed the button color, but i have some problems with the position. Will customize the modal later.